### ISSUE TITLE
Just a question!

### ISSUE THREAD

**Issue by MysticNynja** [tzarc/djinn#3]
Can the left be used independently of the right, say for gaming?

**Comment by tzarc** [comment]
Yep! QMK picked up that functionality a few months ago, I just need to re-sync with upstream.

**Comment by MysticNynja** [comment]
Nice!!!! It's like you read my mind in what I wanted! 

Couple things I might try to add is a tent and an analog stick instead of dpad on left side. Either way off to buy some stuff!!

**Comment by MysticNynja** [comment]
One more question. Any plans for a wireless version?

**Comment by tzarc** [comment]
None at this stage. If/when a wireless design comes along, it'll likely be a whole new keyboard -- charger circuits and the like need integrating.

**Comment by MysticNynja** [comment]
Sounds good to me. Just need to get a longer cable :) won't hurt to build a couple anyway. Tha KS again
